### ISSUE TITLE
better accessibility support

### ISSUE THREAD

**Issue by davelandry** [d3plus/d3plus-viz#82]
The following features should be implemented for better accessibility and 508 compliance:

- [x] https://github.com/d3plus/d3plus-shape/issues/78
- [x] https://github.com/d3plus/d3plus-shape/issues/79
- [x] https://github.com/d3plus/d3plus-text/issues/89
- [x] add a `<title id="${uuid}-title">` tag containing `this._title || "D3plus Visualization"` ([docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) on SVG titles, code will probably need to be inserted [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L418))
- [x] set `ariaHidden: true` on the visible `TextBox` chart title (should be able to add to titleConfig [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L187-L192))
- [x] add a `<desc id="${uuid}-desc">` tag mapped to a new `desc` method and `this._desc` (https://developer.mozilla.org/en-US/docs/Web/SVG/Element/desc) on SVG titles, code will probably need to be inserted [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L418))
- [x] add `aria-labelledby="${this._uuid}-title ${this._uuid}-desc"` to the `<svg>` container (SVG is modified [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L426-L428))
- [x] add `role="img"` to the `<svg>` container (SVG is modified [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L426-L428))
- [x] add `role="presentation"` to all shapes (can probably be added to the shapeConfig [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L150-L180))
- [x] set `ariaLabel` for all shapes to the `drawLabel` (to be overridden by charts as necessary) (can probably be added to the shapeConfig [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L150-L180))
- [x] set `ariaLabel` for all legend shapes to the `legendLabel` (can probably be added to the legendConfig [here](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L100-L109))
